frontend/shared/src/utils/__tests__/date.utils.test.ts

```diff
@@ -39,4 +45,19 @@ describe('dates', () => {
       expect(diffMonthsResult).toBeLessThan(0);
     });
   });
+
+  describe('validateDateIsFromCurrentYearOnwards', () => {
+    it('should return false', () => {
+      const validateDateIsFromCurrentYearOnwardsResult =
+        validateDateIsFromCurrentYearOnwards('31.12.2021');
```

Comment:
`31.12.${new Date().getFullYear()-1}` could more of an edge value in the future too :)
Also If you want to get path coverage 100% then there could be one more test with invalid string value which should return false.

Reply:
Thank you, I have updated the test, if it's ok then I will merge